api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -107,6 +110,20 @@ public void validate(Object target, Errors errors) {
 				validateStopDatetime(visit, otherVisit, errors);
 			}
 		}
+        	if(visit.getStartDatetime()!=null && visit.getPatient()!=null && visit.getPatient().getBirthdate()!=null)
+        	{
+            		Date birthDateTime = visit.getPatient().getBirthdate();
+            		Date startDateTime = visit.getStartDatetime();
+            		if(startDateTime.compareTo(birthDateTime) < 0) {
```

Comment:
Actually I thought we always compared dates that might come from the DB using OpenmrsUtil.compare(Date, Date) because there's sometimes an issue where the date on a hibernate proxy is actually a java.sql.Timestamp (which isn't correctly comparable to a java.util.Date)